Tests/MongoSwiftSyncTests/ClientSessionTests.swift

```diff
@@ -51,13 +63,14 @@ final class ClientSessionTests: MongoSwiftTestCase {
         (name: "deleteOne", body: { _ = try $0.deleteOne([:], session: $1) }),
         (name: "deleteMany", body: { _ = try $0.deleteMany([:], session: $1) }),
         (name: "createIndex", body: { _ = try $0.createIndex([:], session: $1) }),
-        (name: "createIndex1", body: { _ = try $0.createIndex(IndexModel(keys: ["x": 1] as Document), session: $1) }),
-        (name: "createIndexes", body: { _ = try $0.createIndexes([], session: $1) }),
+        (name: "createIndex1", body: { _ = try $0.createIndex(IndexModel(keys: ["x": 1]), session: $1) }),
+        (name: "createIndexes", body: { _ = try $0.createIndexes([IndexModel(keys: ["x": 1])], session: $1) }),
```

Comment:
this was failing because I added a check in `createIndexes` for `models` being non-empty